src/pages/Loans/LoansOverview/components/CollateralModal/CollateralModal.tsx

```diff
@@ -62,11 +60,8 @@ const CollateralModal = ({ asset, position, onClose, ...props }: CollateralModal
   const prices = useGetPrices();
 
   const transaction = useTransaction({
```

Comment:
@nud3l Do we want to block the UI when toggling loans collateral? There are some scenarios that we need to be careful with, for example, if I toggle off a collateral and do it again, im basically going to have the second one failing

![image](https://github.com/interlay/interbtc-ui/assets/43269067/5cecd2d3-0a03-4fbe-ba97-2dc31a409bc7)

is this okay?

Reply:
Yes, let's block in this case.